lib/google_contacts_api/contacts_accessor.rb

```diff
@@ -25,5 +25,37 @@ def get_contacts(params = {})
       end
       GoogleContactsApi::ContactSet.new(response.body, @api)
     end
+
+    def post_contact(contact)
+      params = {}
+      headers = {
+          'Content-type' => 'application/json'
+          }
+      body = contact.json
+      params[:body] = body
+      url = "contacts/default/full"
+      response = @api.post_v2(url, params, headers)
+      GoogleContactsApi::Api.parse_response_code(response)
+    end
+
+    def put_contact(contact)
+      params = {}
+      headers = {
+          'Content-type' => 'application/json',
+          'If-Match' => '*'
+          }
+      body = contact.json
+      params[:body] = body
+      response = @api.put_v2(contact.edit_link, params, headers)
+      GoogleContactsApi::Api.parse_response_code(response)
+    end
+
+    def put_or_post_contact(contact)
```

Comment:
I'm also not sure about this name, I was going with save_contact, but I also called a method in the user class that, so I'm worried about name collision, as that class imports this module